src/client/timer.c

```diff
@@ -0,0 +1,117 @@
+#include "timer.h"
+#include <SDL_ttf.h>
+#include <SDL.h>
+#include <SDL_image.h>
+#include "game.h"
+
+void CreateTimer(Timer *timer)
+{
+    TTF_Init(); // va tvungen att lägga den här för att initiera Sans
```

Comment:
Remove, already done in main.c

Reply:
the game crashes without that initialization

Reply:
It should not, this function is already called in main.c

Reply:
Call it inside the GetGame initializer